src/spec/dsl.cr

```diff
@@ -64,6 +65,28 @@ module Spec
     exit
   end
 
+  # :nodoc:
+  class_getter randomizer_seed : UInt64?
+  class_getter randomizer : Random::PCG32?
+
+  # :nodoc:
+  def self.order=(mode)
+    seed =
+      case mode
+      when "default"
+        nil
+      when "random"
+        Random::Secure.rand(1..99999).to_u64 # 5 digits or less for simplicity
+      when UInt64
+        mode
+      else
+        raise ArgumentError.new("order must be either 'default', 'random', or a seed value")
+      end
+
+    @@randomizer_seed = seed
+    @@randomizer = seed && Random::PCG32.new(seed)
```

Comment:
I'd prefer this initialization in an `if` conditional instead of guarded by a fancy logic expression.

Reply:
This is equivalent to 

```crystal
@@randomizer = seed ? Random::PCG32.new(seed) : nil
```

Is that preferable?